src/pudl/analysis/record_linkage/eia_ferc1_model_config.py

```diff
@@ -33,59 +32,57 @@
     blocking_rule_10,
 ]
 
-plant_name_comparison = ctl.name_comparison(
+plant_name_comparison = cl.NameComparison(
     "plant_name",
-    damerau_levenshtein_thresholds=[],
     jaro_winkler_thresholds=[0.9, 0.8, 0.7],
 )
-utility_name_comparison = ctl.name_comparison(
+utility_name_comparison = cl.NameComparison(
     "utility_name",
-    damerau_levenshtein_thresholds=[],
     jaro_winkler_thresholds=[0.9, 0.8, 0.7],
-    term_frequency_adjustments=True,
-)
-fuel_type_code_pudl_comparison = cl.exact_match(
-    "fuel_type_code_pudl", term_frequency_adjustments=True
 )
+utility_name_comparison.configure(term_frequency_adjustments=True)
+fuel_type_code_pudl_comparison = cl.ExactMatch("fuel_type_code_pudl")
+fuel_type_code_pudl_comparison.configure(term_frequency_adjustments=True)
+
 capacity_comparison = {
     "output_column_name": "capacity_mw",
     "comparison_levels": [
-        cll.null_level("capacity_mw"),
-        cll.percentage_difference_level(
+        cll.NullLevel("capacity_mw"),
+        cll.PercentageDifferenceLevel(
             "capacity_mw",
             0.0 + 1e-4,
         ),
-        cll.percentage_difference_level("capacity_mw", 0.05),
-        cll.percentage_difference_level("capacity_mw", 0.1),
-        cll.percentage_difference_level("capacity_mw", 0.2),
-        cll.else_level(),
+        cll.PercentageDifferenceLevel("capacity_mw", 0.05),
+        cll.PercentageDifferenceLevel("capacity_mw", 0.1),
+        cll.PercentageDifferenceLevel("capacity_mw", 0.2),
+        cll.ElseLevel(),
     ],
     "comparison_description": "0% different vs. 5% different vs. 10% different vs. 20% different vs. anything else",
 }
 
 net_gen_comparison = {
     "output_column_name": "net_generation_mwh",
     "comparison_levels": [
-        cll.null_level("net_generation_mwh"),
-        cll.percentage_difference_level(
+        cll.NullLevel("net_generation_mwh"),
+        cll.PercentageDifferenceLevel(
             "net_generation_mwh", 0.0 + 1e-4
         ),  # could add an exact match level too
-        cll.percentage_difference_level("net_generation_mwh", 0.01),
-        cll.percentage_difference_level("net_generation_mwh", 0.1),
-        cll.percentage_difference_level("net_generation_mwh", 0.2),
-        cll.else_level(),
+        cll.PercentageDifferenceLevel("net_generation_mwh", 0.01),
+        cll.PercentageDifferenceLevel("net_generation_mwh", 0.1),
+        cll.PercentageDifferenceLevel("net_generation_mwh", 0.2),
+        cll.ElseLevel(),
     ],
     "comparison_description": "0% different vs. 1% different vs. 10% different vs. 20% different vs. anything else",
 }
 
 
 def get_date_comparison(column_name):
     """Get date comparison template for column."""
-    return ctl.date_comparison(
+    return cl.DateOfBirthComparison(
         column_name,
-        damerau_levenshtein_thresholds=[],
-        datediff_thresholds=[1, 2],
-        datediff_metrics=["year", "year"],
+        input_is_string=False,
```

Comment:
This `input_is_string=False` parameter is the only thing I was worried about being more than a rename -- it means that the date field that's being matched is a `Date` or `DateTime` type, which I wasn't sure about. But the ETL worked, so I'm assuming it was correct.